### PR TITLE
Frond end design changes

### PR DESCRIPTION
**Pull request by CS76**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# OS generated files #
```
Comment (johnmay) [inline .gitignore]: You should put this in your global .gitignore

**Comment by johnmay** [comment]
Looks good, just double checking did you draw the icons your self? If not are there any restrictions on them?

**Comment by johnmay** [comment]
Also this repo should have the hosted dist/ as the root package.

**Comment by johnmay** [comment]
You need to add attribution for icons. FlatIcons digital marketing set it looks like possibly? Also can you reorganise it such that the pull request is just the dist directory content.

If I pushed it as it is now I would get 'https://cdk.github.io/dist/index.html' and 'https://cdk.github.io/src/index.html' instead of https://cdk.github.io/index.html

Then send the me npm/source dir separately and I'll add another orphan branch.

**Comment by CS76** [comment]
I do have the premium license to use the icons on any website without any attribution. So its should not be a problem.

**Comment by CS76** [comment]
Regarding the dist directory... can you create a gh-pages for the dist directory alone ?? 

``` git subtree push --prefix dist origin gh-pages ```

 and then specify the root branch on GitHub..

File: .gitignore (hunk 1 above)
Comment (CS76) [inline .gitignore]: 👍

**Comment by johnmay** [comment]
gh-pages is used for project pages e.g. cdk.github.com/cdk which is located at github.com/cdk/cdk:gh-pages branch but not sure if that works for this specific project which is name mapped.

Will give it a shot.

**Comment by johnmay** [comment]
As expected: messed up and only shows the README :-)

https://help.github.com/articles/user-organization-and-project-pages/

**Comment by CS76** [comment]
Okie, will reorganise the structure and send the pull request again. How would you like me to send the source ??

**Pull request merged**